Sources/SmithyStreams/StreamableHttpBody.swift

```diff
@@ -109,4 +109,8 @@ public class StreamableHttpBody: IStreamable {
             return nil
         }
     }
+
+    public func isEndOfStream() -> Bool {
+        return false
```

Comment:
What's the rationale for this always being false?

Reply:
Rationale was provided in CRT side [here](https://github.com/awslabs/aws-crt-swift/compare/v0.50.0...v0.51.0#diff-1fa8771c6bca4f8944174b59d9deb27d25e8536e47f7ecd7dad3d3bf9ab8a74dR33-R35).

Reply:
After discussion with Waqar realized it has to have actual impl in SDK side; updated here: https://github.com/smithy-lang/smithy-swift/pull/925/commits/55dd83c014dc630702a84c4d290c32b4cd4ea6e0